oxd-server/src/main/java/org/gluu/oxd/server/Cli.java

```diff
@@ -293,4 +299,35 @@ private static Options options() {
 
         return options;
     }
+
+    private static boolean checkForMultipleOptions(CommandLine cmd){
+        int optionsCount = 0;
+        if (cmd.hasOption("l")) {
+            optionsCount ++;
+        }
+        if (cmd.hasOption("oxd_id")) {
+            optionsCount ++;
+        }
+        if (cmd.hasOption("d")) {
+            optionsCount ++;
+        }
+        if(optionsCount > 1)
+            return true;
+        else
+            return false;
+    }
+
+    private static boolean checkForArgsAfterListParameter(String[] args){
```

Comment:
rename `hasListParameterValue`

Reply:
checkForArgsAfterListParameter renamed to hasListParameterValue.